rootfs-builder/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a rootfs for Kata Containers Guest OS #
+
+Kata Containers guest OS is created using `rootfs.sh`.
+
+## Supported base OS ##
+
+The `rootfs.sh` script support different based OS `rootfs`. To build
+a `rootfs` based in a supported based OS run:
+
+``` ./rootfs.sh <distro> ```
+
+To check the supported `rootfs` based OS use:
```

Comment:
This isn't clear to me. Do you mean that if you run the script with `-h`, it will show the supported values of `<distro>`?